connectors/scripts/generate_connector_inspect.py

```diff
@@ -1,5 +1,27 @@
 import argparse
+import os, requests
+import json, logging, shutil
+
 from connectors.scripts.utils import is_path_exist, create_path, get_dir_name
+from os.path import join
+from pathlib import Path
+from json2html import *
+
+from logging.handlers import RotatingFileHandler
+from camelcase import CamelCase
+from PIL import Image
+
+LOG_FILE_PATH = join(os.path.dirname(os.path.abspath(__file__)), 'unit_test.log')
```

Comment:
Please make sure that logging is consistent with other plugin logs. You can have logger class initiated with different file name here.